src/_gettsim_tests/test_vorsorgeaufw.py

```diff
@@ -23,7 +23,7 @@
 ]
 OUT_COLS = ["vorsorgeaufw_tu"]
 
-YEARS = [2004, 2005, 2010, 2018, 2020, 2021, 2022, 2024, 2025]
+YEARS = [2004, 2005, 2010, 2018, 2020, 2021, 2022, 2025]
```

Comment:
```suggestion
YEARS = [2004, 2005, 2010, 2018, 2020, 2021, 2022, 2023]
```
? 

Seems important to make sure the discrete change is caught correctly at the right time?